Source/Core/DolphinLib.props

```diff
@@ -531,7 +531,7 @@
     <ClInclude Include="VideoBackends\D3DCommon\D3DCommon.h" />
     <ClInclude Include="VideoBackends\D3DCommon\Shader.h" />
     <ClInclude Include="VideoBackends\D3DCommon\SwapChain.h" />
-    <ClInclude Include="VideoBackends\Null\NullPerfQuery.h" />
```

Comment:
Is removing NullPerfQuery.h a mistake?

Reply:
Nope, it's a non-existent file. Must've been leftover accidentally from some past refactor.